Linkbus/Linkbus/RouteController.swift

```diff
@@ -269,7 +273,7 @@ extension RouteController {
             refreshedLbBusSchedule.alerts.append(dailyMessageAlert)
         }
         
-        if !(csbsjuApiResponse.routes!.isEmpty) {
+        if (!csbsjuApiResponse.routes!.isEmpty) {
```

Comment:
This didn't actually change anything right? Just preference?

Reply:
Accident I suppose. You can't do that in JavaScript so it hurts my brain to see.